Sources/TokamakCore/MountedViews/MountedElement.swift

```diff
@@ -217,14 +234,15 @@ extension TypeInfo {
 extension AnyView {
   func makeMountedView<R: Renderer>(
     _ parentTarget: R.TargetType,
-    _ environmentValues: EnvironmentValues
+    _ environmentValues: EnvironmentValues,
+    _ parent: MountedElement<R>?
```

Comment:
Do we need to be explicit with this, or would `nil` as a default argument make sense here?
```suggestion
    _ parent: MountedElement<R>? = nil
```

Reply:
I guess the argument for no default argument would be that only the root element (when creating the `StackReconciler`) sets it to `nil`. So having it be explicit would catch accidental missed arguments.